### PR TITLE
feat: extend user with more props and team information

### PR DESCRIPTION
**Pull request by fampinheiro**

### CODE REVIEW

File: packages/model/src/user.ts [hunk 1]
```diff
@@ -4,11 +4,18 @@ export interface Invitee {
...
+  jobTitle?: string;
   orcid?: string;
+  department?: string;
   institution?: string;
+  location?: string;
 }
 
 export interface UserResponse extends Invitee {
   id: string;
+  teams: {
```
Comment (johnytiago) [inline packages/model/src/user.ts:16]: What do you think of re-using the TeamMember type https://github.com/yldio/asap-hub/blob/master/packages/model/src/team.ts#L1

Comment (fampinheiro) [reply packages/model/src/user.ts:16]: I understand the reasoning, but this is just an happy coincidence. I don't expect them to evolve in the same way.

Comment (johnytiago) [reply packages/model/src/user.ts:16]: Do you expect to return different data if you fetch team members or if you fetch team affiliations?

Comment (fampinheiro) [reply packages/model/src/user.ts:16]: I'm expecting the team information when you get `/users/:id` to be the short form (`id`, `displayName`, `role`) but when fetching teams `/teams/:id` the user may have more information.

**Pull request merged**